Cargo.toml

```diff
@@ -1,9 +1,22 @@
 [workspace]
 members = [
-    "opentelemetry-*",
+    "opentelemetry-aws",
```

Comment:
Why change the wildcard pattern?

Reply:
It's mentioned here - https://github.com/open-telemetry/opentelemetry-rust-contrib/pull/190#discussion_r1980656596